app/ts/Elicitation/ImpreciseSwingElicitation/ImpreciseSwingButtons/ImpreciseSwingButtons.tsx

```diff
@@ -0,0 +1,80 @@
+import Button from '@material-ui/core/Button';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import {ElicitationContext} from 'app/ts/Elicitation/ElicitationContext';
+import {PreferencesContext} from 'app/ts/Elicitation/PreferencesContext';
+import _ from 'lodash';
+import React, {useContext} from 'react';
+
+export default function ImpreciseSwingButtons() {
```

Comment:
Isn't this file (almost) the same for precise swing? Can they be combined into one?